src/zkp/piaffg.rs

```diff
@@ -291,6 +271,38 @@ impl Proof for PiAffgProof {
     }
 }
 
+impl PiAffgProof {
+    // trunk-ignore(clippy/too_many_arguments)
+    fn fill_out_transcript(
```

Comment:
@marsella this method has too many argument, what do you recommend doing ?

Reply:
Ugh. #[allow(clippy::too_many_arguments)] and add a comment in #44 to look into removing it.